src/main/kotlin/net/irisshaders/lilybot/utils/DatabaseHelper.kt

```diff
@@ -537,6 +537,94 @@ object DatabaseHelper {
 		val collection = database.getCollection<RemindMeData>()
 		return collection.find().toList()
 	}
+
+	/**
+	 * Set's a channel as blacklisted for uploading logs.
+	 *
+	 * @param inputGuildId The guild the command was run in
+	 * @param inputChannelId The channel to disable uploading for
+	 *
+	 * @author NoComment1105
+	 * @since 3.5.4
+	 */
+	suspend inline fun setLogUploadingBlacklist(inputGuildId: Snowflake, inputChannelId: Snowflake) {
+		val collection = database.getCollection<LogUploadingBlacklistData>()
+		collection.insertOne(LogUploadingBlacklistData(inputGuildId, inputChannelId))
+	}
+
+	/**
+	 * Removes a channel from the blacklist for uploading logs.
+	 *
+	 * @param inputGuildId The guild the command was run in
+	 * @param inputChannelId The channel to re-enable uploading for
+	 *
+	 * @author NoComment1105
+	 * @since 3.5.4
+	 */
+	suspend inline fun removeLogUploadingBlacklist(inputGuildId: Snowflake, inputChannelId: Snowflake) {
```

Comment:
You should only need the channel ID for this.

Reply:
Better to be safe imo